src/ORM/Table.php

```diff
@@ -1011,6 +1011,14 @@ public function findAll(Query $query, array $options)
      */
     public function findList(Query $query, array $options)
     {
+        if (!isset($options['keyField']) && !isset($options['valueField'])) {
+            $select = $query->clause('select');
+            if ($select && count($select) <= 2) {
+                $options['keyField'] = array_shift($select);
+                $options['valueField'] = array_shift($select) ?: $options['keyField'];
```

Comment:
What happens when the field is in he form of "table.field"? What if the user tries to select a field from another table?